### ISSUE TITLE
Add guide for Palemoon netscape theme

### ISSUE THREAD

**Issue by DarkReaper231** [grassmunk/Chicago95#317]
Palemoon is probably the only browser out there that perfectly coexists with Chicago95

![image](https://github.com/grassmunk/Chicago95/assets/38611351/fd1f2ba5-a1e2-424d-9043-a37433a13975)

It's pretty simple to pull off, just install this addon: https://addons.palemoon.org/addon/moonscape/

I would also like to donate the homepage I made for this, you can add that to the repo as well
[homepage.tar.gz](https://github.com/grassmunk/Chicago95/files/13543196/homepage.tar.gz)

**Comment by grassmunk** [comment]
I agree, do you mind adding this to the documentation and making a pull request?
